terraform/install.sh

```diff
@@ -232,7 +232,7 @@ installMonitoring() {
   log "Creating monitoring examples (dashboards, uptime checks, alerting policies, etc.)..."
   pushd monitoring/
   terraform init -lock=false
-  terraform apply --auto-approve -var="project_id=${project_id}" -var="external_ip=${external_ip}" -var="project_owner_email=${acct}"
+  terraform apply --auto-approve -var="project_id=${project_id}" -var="external_ip=${external_ip}" -var="project_owner_email=${acct}" -var="zone=${CLUSTER_ZONE}"
```

Comment:
Does this mean that install instructions need to be changed to ensure the user sets this before doing the install or is this env variable already defined?

Reply:
Nope install instructions remain the same! The env variable is already defined